wgsl/index.bs

```diff
@@ -6086,7 +6084,24 @@ TODO: *Stub*
   * Initializers evaluated in textual order
 * No two variables have overlapping storage (might already be covered earlier?)
 
-### Program order (within an invocation) TODO ### {#program-order}
+### Program order (within an invocation) ### {#program-order}
+
+Each statement in a [SHORTNAME] program may be executed zero or more times during
+execution.
```

Comment:
Does this need saying? The only numbers of times executed it excludes are negative values?

Reply:
I was just trying to specifically call out that not all statements will necessarily be executed.